### PR TITLE
Standalone HA Updates & Test

### PR DESCRIPTION
**Pull request by cicoyle**
- Set all etcd urls: peer & client and set for etcd server the client urls for communication to be successful
- Ensure if I schedule a job on 1 scheduler instance that I can see that job reflected in the other schedulers for standalone mode
- make initialCluster a []string to match placement input format
- make lint
- EtcdClientPorts is now a []string of `schedulerID=clientPort`, this is needed bc I have to pass along the ip & client ports for all scheduler instances (etcd servers) to the cron client lib for the mutex builder

Since the library GetJob only checks from the in memory list, I am running an `etcdctl` command. This should be removed and swapped for the GetJob logic once that checks from the db.

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: tests/integration/suite/scheduler/api/jobs.go [hunk 1]
```diff
@@ -63,12 +77,12 @@ func (j *jobs) Run(t *testing.T, ctx context.Context) {
...
 				Job: &rtv1.Job{
 					Name:     name,
-					Schedule: "@daily",
+					Schedule: "@every 1s",
 					Data: &anypb.Any{
 						Value: []byte("test"),
 					},
-					Repeats: 1,
-					DueTime: "0h0m9s0ms",
+					Repeats: 10,
```
Comment (artursouza) [inline tests/integration/suite/scheduler/api/jobs.go]: We don't have this quite yet.

File: tests/integration/suite/scheduler/quorum/notls.go [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+		diffEtcdCmdOutput := executeEtcdCmd(t, diffSchedulerPort)
+		checkCmdOutputForAppID(t, diffEtcdCmdOutput)
+	}
+}
+
+// TODO: Cassie - rm this and switch to the get func once the get functionality checks from the db
+// as of now, the get func only checks in memory, so the data wont be there on another scheduler instance
+func executeEtcdCmd(t *testing.T, port string) string {
+	cmd := exec.Command("etcdctl", "get", "", "--prefix", fmt.Sprintf("--endpoints=localhost:%s", port))
```
Comment (artursouza) [inline tests/integration/suite/scheduler/quorum/notls.go]: Please don't require etcdctl, you can use the etcd's go library instead.

**Review by artursouza — COMMENTED** [review]
Minor comments.

File: tests/integration/suite/scheduler/quorum/notls.go (hunk 2 above)
Comment (cicoyle) [reply tests/integration/suite/scheduler/quorum/notls.go]: Oh, let me look into that

File: tests/integration/suite/scheduler/api/jobs.go (hunk 1 above)
Comment (cicoyle) [reply tests/integration/suite/scheduler/api/jobs.go]: I figured the more we had now the less we had to add later, but I can rm it for now.

Comment (artursouza) [reply tests/integration/suite/scheduler/api/jobs.go]: That is OK.

Comment (artursouza) [reply tests/integration/suite/scheduler/api/jobs.go]: As long as your test does not require it.

**Review by artursouza — APPROVED** [review]

**Pull request merged**